packages/teleterm/src/ui/services/workspacesService/workspacesService.ts

```diff
@@ -210,7 +210,18 @@ export class WorkspacesService extends ImmutableStore<WorkspacesState> {
           })
             ? {
                 location: persistedWorkspace.location,
-                documents: persistedWorkspaceDocuments,
+                documents: persistedWorkspaceDocuments.map(d => {
+                  if (
+                    d.kind === 'doc.terminal_tsh_node' ||
+                    d.kind === 'doc.gateway'
+                  ) {
+                    return {
+                      ...d,
+                      status: 'disconnected',
+                    };
+                  }
+                  return d;
+                }),
```

Comment:
What's the purpose behind this change?

No matter the purpose though, adding code like this is tricky because then each item we add a new document that should act kinda like `doc.terminal_tsh_node` or `doc.gateway`, we'll have checks like this one scattered across the codebase.

It'd be better to add a function to a file related to documents, which at the moment we don't have I believe.

Still, a lone function that can be imported defined in `documentsService.ts` would be better then this, or maybe we could even have `documentsService/document.ts` with functions that operate on `Document`.

Reply:
I decided to remove it from the PR as it doesn't really solve the issue. I wanted to show `Reconnect` screen when the cluster is offline, but we don't keep the information about that state. 
Basically, `useReconnect` will make this check:
```
useEffect(() => {
    if (cluster.connected) {
      markDocumentAsConnected();
    }
  }, []);
```
and change the document state to `connected`.

Let's leave it as it is for now.